src/battle_game/core/units/double_scatter_tank.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+#include "battle_game/core/unit.h"
+
+namespace battle_game::unit {
+class DoubleScatterTank : public Unit {
+ public:
+  DoubleScatterTank(GameCore *game_core, uint32_t id, uint32_t player_id);
+  void Render() override;
+  void Update() override;
```

Comment:
Actually, you can inherit from unit::Tank instead of Unit to avoid adding these repeated codes.

Reply:
Done.